addons/vehicles_dragoon/CfgWeapons.hpp

```diff
@@ -0,0 +1,159 @@
+class CfgWeapons {
+    class CannonCore;
+    class autocannon_Base_F: CannonCore {
+        class player;
+    };
+    class autocannon_30mm_CTWS: autocannon_Base_F
+    {
+        displayName = "$STR_A3_GATLING_30MM_CTWS0";
+        muzzles[] = {"HE","AP"};
+        class HE: autocannon_Base_F
+        {
+            displayName = "$STR_A3_GATLING_30MM_CTWS0";
+            magazines[] = {"140Rnd_30mm_MP_shells","140Rnd_30mm_MP_shells_Tracer_Red","140Rnd_30mm_MP_shells_Tracer_Green","140Rnd_30mm_MP_shells_Tracer_Yellow","78Rnd_30mm_MP_shells_Tracer_Red","78Rnd_30mm_HEI_shells_Tracer_Red"};
+            magazineReloadTime = 40;
+            class player: player
+            {
+                dispersion = 0.0018;
```

Comment:
Include only values we change.
Due to changing this turret other turrets using this cannon will also have the effect you add here.